### PR TITLE
unused param in Service::OpenSession

### PR DESCRIPTION
**Pull request by chenxiaofa**
missing param username

### CODE REVIEW

File: src/hs2client/service.cc [hunk 1]
```diff
@@ -78,7 +78,7 @@ void Service::SetSendTimeout(int timeout) {
 Status Service::OpenSession(const string& user, const HS2ClientConfig& config,
     unique_ptr<Session>* session) const {
   session->reset(new Session(rpc_));
-  return (*session)->Open(config);
+  return (*session)->Open(config,user);
```
Comment (mjacobs) [inline src/hs2client/service.cc]: nit: space between config, and user

File: src/hs2client/session.cc [hunk 2]
```diff
@@ -50,9 +50,10 @@ Status Session::Close() {
   return TStatusToStatus(resp.status);
 }
 
-Status Session::Open(const HS2ClientConfig& config) {
+Status Session::Open(const HS2ClientConfig& config,const string& user) {
```
Comment (mjacobs) [inline src/hs2client/session.cc]: nit: space again after the comma

File: src/hs2client/session.h [hunk 3]
```diff
@@ -62,7 +62,7 @@ class Session {
 
   // Performs the RPC that initiates the session and stores the returned handle.
   // Must be called before operations can be executed.
-  Status Open(const HS2ClientConfig& config);
+  Status Open(const HS2ClientConfig& config,const string& user);
```
Comment (mjacobs) [inline src/hs2client/session.h]: nit: space after comma

File: src/hs2client/session.h [hunk 4]
```diff
@@ -20,7 +20,7 @@
 #include "hs2client/service.h"
 #include "hs2client/macros.h"
 #include "hs2client/operation.h"
-
+using std::string;
```
Comment (mjacobs) [inline src/hs2client/session.h]: avoid 'using' in header files, just use std::string below

**Comment by chenxiaofa** [comment]
changes commmitted

**Pull request merged**